<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="130797" started-at="2018-07-29T10:12:43Z" finished-at="2018-07-29T10:14:54Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="130797" started-at="2018-07-29T10:12:43Z" finished-at="2018-07-29T10:14:54Z">
      <class name="practiceTest.Login">
        <test-method status="PASS" signature="setExtent()[pri:0, instance:practiceTest.Login@6a41eaa2]" name="setExtent" is-config="true" duration-ms="1171" started-at="2018-07-29T10:12:43Z" finished-at="2018-07-29T10:12:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:practiceTest.Login@6a41eaa2]" name="setUp" is-config="true" duration-ms="72023" started-at="2018-07-29T10:12:45Z" finished-at="2018-07-29T10:13:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="openBrowser()[pri:1, instance:practiceTest.Login@6a41eaa2]" name="openBrowser" duration-ms="42128" started-at="2018-07-29T10:13:57Z" finished-at="2018-07-29T10:14:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:practiceTest.Login@6a41eaa2]" name="tearDown" is-config="true" duration-ms="1379" started-at="2018-07-29T10:14:39Z" finished-at="2018-07-29T10:14:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=openBrowser status=SUCCESS method=Login.openBrowser()[pri:1, instance:practiceTest.Login@6a41eaa2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:practiceTest.Login@6a41eaa2]" name="setUp" is-config="true" duration-ms="11362" started-at="2018-07-29T10:14:40Z" finished-at="2018-07-29T10:14:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="titleTest()[pri:2, instance:practiceTest.Login@6a41eaa2]" name="titleTest" duration-ms="41" started-at="2018-07-29T10:14:51Z" finished-at="2018-07-29T10:14:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [google1234] but found [Google]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [google1234] but found [Google]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at practiceTest.Login.titleTest(Login.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:practiceTest.Login@6a41eaa2]" name="tearDown" is-config="true" duration-ms="1704" started-at="2018-07-29T10:14:52Z" finished-at="2018-07-29T10:14:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=titleTest status=FAILURE method=Login.titleTest()[pri:2, instance:practiceTest.Login@6a41eaa2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:practiceTest.Login@6a41eaa2]" name="endReport" is-config="true" duration-ms="865" started-at="2018-07-29T10:14:53Z" finished-at="2018-07-29T10:14:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- practiceTest.Login -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
